pkg/operator/generic_client.go

```diff
@@ -147,7 +148,6 @@ func (p *genericClient) ApplyOperatorStatus(ctx context.Context, fieldManager st
 		desiredStatus := &v1.OpenShiftControllerManagerStatusApplyConfiguration{
 			OperatorStatusApplyConfiguration: *desiredConfiguration,
 		}
-		desired := v1.OpenShiftControllerManager("cluster")
 		desired.WithStatus(desiredStatus)
 		_, err = p.client.OpenShiftControllerManagers().ApplyStatus(ctx, desired, metav1.ApplyOptions{
```

Comment:
This line is duplicated as well

Reply:
yeah, good catch. I forgot to remove this and if you didn't catch, we'd execute ApplyStatus twice. Thank you.